easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['nest-%(version)s.tar.gz']
+source_urls = ['https://github.com/nest/nest-simulator/releases/download/v%(version)s/']
```

Comment:
Please include a `SHA256` checksum.

With EasyBuild v3.4.0, `eb` can do that for you, using `eb NEST-2.12.0-foss-2015b.eb --inject-checksums`, see also http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#adding-or-replacing-checksums-using-inject-checksums